ql/src/java/org/apache/hadoop/hive/ql/cache/results/QueryResultsCache.java

```diff
@@ -989,7 +990,7 @@ public void accept(NotificationEvent event) {
       QueryResultsCache cache = QueryResultsCache.getInstance();
       if (cache != null) {
         long eventTime = event.getEventTime() * 1000L;
-        cache.notifyTableChanged(dbName, tableName, eventTime);
+        cache.notifyTableChanged(TableName.fromString(tableName, null, dbName), eventTime);
```

Comment:
I wonder if it would make sense to leave out the category for now....it's not really used - instead of passing null everywhere we could have a separate method for (db,name)